src/modules/workspaces/workspaces.controller.ts

```diff
@@ -348,4 +349,24 @@ export class WorkspacesController {
       changeUserRoleBody,
     );
   }
+  @Patch('/:workspaceId')
+  @ApiOperation({
+    summary: 'Edit workspace details',
+  })
+  @ApiBearerAuth()
+  @ApiParam({ name: 'workspaceId', type: String, required: true })
+  @UseGuards(WorkspaceGuard)
+  @WorkspaceRequiredAccess(AccessContext.WORKSPACE, WorkspaceRole.OWNER)
+  async editWorkspaceDetails(
```

Comment:
If you use `async`, `await` the call. If not, this keyword is not needed

Reply:
Either return directly and remove async-await or use async-await and store the result in a variable and return afterward